### ISSUE TITLE
`sap-cloud-sdk add-cx-server` produces broken `cx-server` script

### ISSUE THREAD

**Issue by fwilhe** [SAP/cloud-sdk-cli#104]
**Describe The Bug**

The script file `cx-server/cx-server` is incomplet and fails to execute after running

```
sap-cloud-sdk init
sap-cloud-sdk add-cx-server
```

**To Reproduce**

```
➜  foo sap-cloud-sdk init
The target directory (.) does not contain a `package.json.`
Should a new `nest.js` project be initialized in the target directory? (y|n): y
Building application scaffold... done
Enter project name (for use in manifest.yml) [foo]:
Do you want to provide anonymous usage analytics to help us improve the SDK? (y|n): n
  ✔ Creating files
  ✔ Modifying test config
  ✔ Adding dependencies to package.json
  ✔ Installing dependencies
  ✔ Modifying `.gitignore`

+--------------------------------------------------------------+
 ✅ Init finished successfully.

 🚀 Next steps:
 - Run the application locally (`npm run start:dev`)
 - Deploy your application (`npm run deploy`)

 🔨 Consider setting up Jenkins to continuously build your app.
 Use `sap-cloud-sdk add-cx-server` to create the setup script.
+--------------------------------------------------------------+
sap-cloud-sdk init  52.76s user 31.70s system 59% cpu 2:22.84 total
➜  foo git:(master) ✗ sap-cloud-sdk add-cx-server
  ✔ Finding potential conflicts
  ✔ Creating files
➜  foo git:(master) ✗ cat cx-server/cx-server
ning
{
    docker run "${docker_arguments[@]}" ppiper/cx-server-companion /cx-server/cx-server-companion.sh "$@"
    exit $?
}
➜  foo git:(master) ✗ cd cx-server
➜  cx-server git:(master) ✗ chmod +x cx-server
➜  cx-server git:(master) ✗ ./cx-server
./cx-server: line 1: ning: command not found
dos2unix: can't open '/cx-server/mount/server.cfg': No such file or directory
/cx-server/cx-server-companion.sh: line 838: /cx-server/mount/server.cfg: No such file or directory
[Error] Failed to load config from server.cfg file.
➜  cx-server git:(master) ✗ sap-cloud-sdk --version
@sap-cloud-sdk/cli/0.1.10 darwin-x64 node-v14.5.0
➜  cx-server git:(master) ✗
```

**Expected Behavior**

The script should should be the file from [here](https://github.com/SAP/devops-docker-cx-server/blob/master/cx-server-companion/life-cycle-scripts/cx-server)

**Used Versions:**

 - `node -v`: v14.5.0
 - `npm -v`: 6.14.5
 - `sap-cloud-sdk -v`: @sap-cloud-sdk/cli/0.1.10 darwin-x64 node-v14.5.0

**Additional Context**

**Comment by FrankEssenberger** [comment]
Thank you for reporting this bug. We will have a look into the issue soon.

**Comment by FrankEssenberger** [comment]
We created a fix and released it in version 0.1.11. Issue should be fixed now.
